<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eol_window</class>
 <widget class="QWidget" name="eol_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="table_set_groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>表传输类型选择</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="file_name_lineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Azi_table.csv</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="file_sel_pushButton">
            <property name="text">
             <string>选择文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="file_size_lineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="kb_label">
            <property name="text">
             <string>KB</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="add_list_pushButton">
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clear_list_pushButton">
            <property name="text">
             <string>清空列表</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>表类型：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="table_type_comboBox">
            <item>
             <property name="text">
              <string>方位导向矢量表</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>俯仰导向矢量表</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>天线位置表</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="data_type_label">
            <property name="text">
             <string>数据格式：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="data_type_comboBox">
            <item>
             <property name="text">
              <string>加特兰CFX28位数据格式</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>32位浮点类型复数</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16位整型复数</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>32位浮点类型</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="ready_transfer_list_groupBox">
        <property name="title">
         <string>传输列表</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="QLabel" name="transfer_list_val_label">
           <property name="text">
            <string>null</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="switch_groupBox">
     <property name="title">
      <string>传输使能</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="upload_pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>上载</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="update_pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>传输</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="state_groupBox">
     <property name="title">
      <string>传输状态</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="0" colspan="2">
       <widget class="QGroupBox" name="transfer_groupBox">
        <property name="title">
         <string>传输进度</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QProgressBar" name="transfer_progressBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="error_num_label">
          <property name="text">
           <string>错误次数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="error_num_val_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="timer_cnt_groupBox">
        <property name="title">
         <string>传输耗时</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="time_cnt_label">
             <property name="text">
              <string>耗时：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="time_cnt_val_label">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="transfer_bytes_groupBox">
        <property name="title">
         <string>传输数据量</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="frames_label">
               <property name="text">
                <string>帧数：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="frame_lcdNumber"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="bytes_label">
               <property name="text">
                <string>字节数：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="bytes_lcdNumber"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="1">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
